Product.API.Service/Program.cs

```diff
@@ -0,0 +1,31 @@
+using Microsoft.EntityFrameworkCore;
```

Comment:
Namespace should be only contain API because we will have a different service layer